migrations/20210628155200_add-claim-rejectreasonid.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.table('Claim', function (table) {
+    table.integer('RejectionReasonId').unsigned()
+  })
+    .catch(function (error) {
+      console.log(error)
+      throw error
+    })
+}
+
+exports.down = function (knex, Promise) {
+  return knex.schema.table('Claim', function (table) {
+    table.dropColumn('RejectionReasonId')
+  })
+    .catch(function (error) {
```

Comment:
do you need all these catches? We tend not to bother as knex will generally just log the error anyway 🤷

Reply:
To be honest Andy, I'm just cutting and pasting current code. There is a lot of post CP move tech debt to deal with like this :)